.lintstagedrc

```diff
@@ -1,3 +1,9 @@
 {
-  "*.{ts,tsx,js,jsx,json,css,md}": ["prettier --write", "git add"]
```

Comment:
Do we still need this `git add` here?

Reply:
`lint-staged` [v10](https://github.com/lint-staged/lint-staged#v10) onwards we do not need to add git add. In our project we are using `"^10.2.0"`. It should take care of this on its own

Reply:
Good to know 👍